PlacenoteSDK.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
     s.name              = 'PlacenoteSDK'
-    s.version           = '1.6.12'
+    s.version           = '1.7'
     s.summary           = 'Placenote SDK brings persistence to AR apps.'
```

Comment:
Change this to - "Build apps that interact with the real world"

for consistency